src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/__init__.py

```diff
@@ -13,15 +13,15 @@ class BotServiceCommandsLoader(AzCommandsLoader):
     def __init__(self, cli_ctx=None):
         from azure.cli.core.commands import CliCommandType
         custom_type = CliCommandType(
-            operations_tmpl='azure.cli.command_modules.botservice.custom#{}',
+            operations_tmpl='azure.cli.command_modules.botservice.bot_operations#{}',
             client_factory=get_botservice_management_client)
         super(BotServiceCommandsLoader, self).__init__(cli_ctx=cli_ctx,
                                                        custom_command_type=custom_type,
                                                        min_profile='2017-03-10-profile',
                                                        suppress_extension=ModExtensionSuppress(
                                                            __name__,
                                                            'botservice',
-                                                           '0.4.0',
+                                                           '0.4.1',
```

Comment:
Why are you bumping the version of the extension that was suppressed?

Reply:
Right now we have a 0.4.1 extension. When this release of the core CLI goes out, we want that extension version to be deprecated. And we want that on machines that have 0.4.1 extension installed, once they update the CLI and they run az bot, it runs from the core CLI and not the extension.

Does that make sense to you @tjprescott ? Basically we want to prioritize the core cli since we don't plan to work wiht extensions anymore and focus on core CLI moving forward.